assets/js/theme/search.js

```diff
@@ -79,6 +86,57 @@ export default class Search extends CatalogPage {
         urlUtils.goToUrl(url);
     }
 
+    activateTab($tabToActivate) {
+        const $tabsCollection = $('[data-search-page-tabs]').find('[role="tab"]');
+
+        $tabsCollection.each((idx, tab) => {
+            const $tab = $(tab);
+
+            if ($tab.is($tabToActivate)) {
+                $tab.removeAttr('tabindex');
+                $tab.attr('aria-selected', true);
+                const activeTabPanelSelector = `#${$tab.attr('aria-controls')}`;
+                $(activeTabPanelSelector).attr('tabindex', 0);
+                return;
+            }
+
+            $tab.attr('tabindex', '-1');
+            $tab.attr('aria-selected', false);
+            const inactiveTabPanelSelector = `#${$tab.attr('aria-controls')}`;
+            $(inactiveTabPanelSelector).removeAttr('tabindex');
+        });
+    }
+
+    onTabChangeWithArrows(event) {
+        const eventKey = event.which;
+        const isLeftOrRightArrowKeydown = eventKey === leftArrowKey
+            || eventKey === rightArrowKey;
+        if (!isLeftOrRightArrowKeydown) return;
+
+        const $tabsCollection = $('[data-search-page-tabs]').find('[role="tab"]');
+
+        const isActiveElementNotTab = $tabsCollection.index($(document.activeElement)) === -1;
+        if (isActiveElementNotTab) return;
+
+        const activeElementId = `#${document.activeElement.id}`;
```

Comment:
This var also should be removed

Reply:
The same - self-declared code > comments

Reply:
after conversation we decided to have variable for if statement

Reply:
Correct, so isActiveElementNotTab should be left but activeElementId should be removed

Reply:
Fair. Done